capeinfra/swimlanes/private.py

```diff
@@ -38,6 +38,17 @@ def __init__(self, name, *args, **kwargs):
             opts=ResourceOptions(parent=self),
         )
 
+        subnet_name = f"{name}-vm-subnet"
```

Comment:
we're probably going to end up with a few subnets. i'd say this one is geared toward compute (even more specifically analysis pipeline compute), so can we name it a little more specifically (the name, name variable and description)?

Reply:
Revised in latest commit, let me know what you think